ibm/mas_devops/roles/turbonomic/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# Define where AppConnect will be installed, the default values below
```

Comment:
Appconnect?

Reply:
@andrercm @whitfiea - Turbonomics will now be hosted on a SaaS environment.  I do not believe we will need to install Turbo, just Kubeturbo.  Should we just delete the Turbonomic code section?

Reply:
@pseemung If it isn't needed then we don't need the role. I am not sure if a customer could use it?

Reply:
@whitfiea I assume a customer should be able to install turbo.  They would need a software license file to activate the software. Not sure who would make the call to keep it or remove it.

Reply:
@pseemung If there is no current requirement to have this role then lets remove it and keep the kubeturbo.